### PR TITLE
Ember cli/init widget popover

### PR DESCRIPTION
**Pull request by dreuss**
Adds a custom bootstrap popover event when a widget in the list view is clicked on.  Upon hover out, the popover dis-appears after a timeout.  This PR is dependent upon 2 other PR's

https://github.com/G5/orion-cms/pull/10
https://github.com/G5/g5-widget-garden/pull/461/files

### CODE REVIEW

File: spec/integration/location_page_spec.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 require "spec_helper"
 
-describe "Integration '/:website_slug/:web_page_template_slug/edit'",
+describe "Integration '/:website_slug/:web_page_template_slug'",
```
Comment (levibrown) [inline spec/integration/location_page_spec.rb]: the one with /edit id the right one

File: spec/integration/location_page_spec.rb [hunk 2]
```diff
@@ -176,6 +176,32 @@
       end
     end
 
+    describe "diz haz da tooltipz" do
+      before do
+        visit "/#{@website.slug}/#{@web_page_template.slug}"
```
Comment (levibrown) [inline spec/integration/location_page_spec.rb]: you will need to add /edit

File: spec/integration/location_page_spec.rb [hunk 3]
```diff
@@ -176,6 +176,32 @@
       end
     end
 
+    describe "diz haz da tooltipz" do
+      before do
+        visit "/#{@website.slug}/#{@web_page_template.slug}/edit"
+        open_gardens
+        # puts current_url
```
Comment (levibrown) [inline spec/integration/location_page_spec.rb]: cleanup

**Comment by levibrown** [comment]
@dreuss you should merge master in to your branch in G5/orion-cms, then update it here

**Comment by dreuss** [comment]
Closing this out, as @thescubageek has merged this into his own branch (liquid-hooks) so this can get shut down.

**Pull request closed without merging**